presto-hive/src/test/java/com/facebook/presto/hive/metastore/InMemoryHiveMetastore.java

```diff
@@ -1,713 +0,0 @@
-/*
```

Comment:
Extract separate PR if not yet in master.

Reply:
this commit was dropped, as someone started to use this